### ISSUE TITLE
A11y issues we can address

### ISSUE THREAD

**Issue by rkzel** [AutarkLabs/open-enterprise#1871]
There are 3 main classes of a11y related problems highlighted by react-axe module:
1. Issues in our code that could have been solved - mostly already dealt with,
2. Aragon UI issues needing mutual agreement,
3. UI issues we can fix on our own.

This ticket is for the third case: [serious: Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI) - below I listed all cases I found in our applications, for the record:

1. Dot Voting: blue number of "dots remaining":

![Screenshot from 2020-01-09 01-13-00.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/8a7111c4-9739-4a19-99ac-70a393621ab0)

2. Dot Voting and Projects, status of vote/application/work submission, both Rejected and Approved:

![Screenshot from 2020-01-10 01-23-20.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/ea7eeb77-9356-455c-bb36-114c444cc84d)

3. Dot Voting and Projects, links to actions like "clear search":

![Screenshot from 2020-01-09 01-20-00.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/2599bc25-5ce5-455f-9923-e1abc786586a)

4. Projects FilterBar, "clear" and active filter tile `[aragon x]` is also highlighted as insufficient:

![Screenshot from 2020-01-09 17-15-45.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/925a2cab-9841-4008-805e-776024889843)

5. Activities in Issue Details log:

![Screenshot from 2020-01-10 01-33-45.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/c4c5951c-ffb6-4e92-85f1-7d98c9dc4ba1)

6. User's GH name in "Logged in":

![Screenshot from 2020-01-09 16-40-24.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/f83d42ab-0661-491e-bb84-afd11b3a33c2)

7. Bounty Widget, the link:

![Screenshot from 2020-01-10 01-33-31.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/7c090a6b-3c60-4d54-a656-cb71c74c08a2)

8. Link to issue in GH, in Issue Card and all Panels:

![Screenshot from 2020-01-09 18-02-25.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/1dcaf6f2-a78a-47b0-ba0d-28889b28be80)

9. Amount label (`2 ETH`) contrast in Issue Card and Details:

![Screenshot from 2020-01-09 18-17-11.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/e1762b01-5e8b-4831-b46c-484238f3ea84)

10. Titles of review buttons in Review panels:

![Screenshot from 2020-01-10 01-22-16.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/3f85af73-c60a-4952-8858-ea41bfc5600f)
